### PR TITLE
Add more parameters to UserContext class

### PR DESCRIPTION
**Pull request by dfvt**
Add more parameters to UserContext

### CODE REVIEW

File: app/src/main/java/corp/kairos/adamastor/UserContext.java [hunk 1]
```diff
@@ -15,17 +21,35 @@
...
     }
-
     public String getContextName() {
         return contextName;
     }
 
     public List<AppDetail> getContextApps() {
         return contextApps;
     }
+    private void removeApp(AppDetail app){
```
Comment (martinhoaragao) [inline app/src/main/java/corp/kairos/adamastor/UserContext.java]: Considering there now is a method to remove an app, any reason for not having one that can add?

I'd rather have someone else review as well before merging.

**Review by martinhoaragao — APPROVED** [review]

**Review by jlsilva1994 — APPROVED** [review]
it's all ok to me!

File: app/src/main/java/corp/kairos/adamastor/UserContext.java [hunk 2]
```diff
@@ -15,17 +21,49 @@
...
     }
-
     public String getContextName() {
         return contextName;
     }
 
     public List<AppDetail> getContextApps() {
         return contextApps;
     }
+    private void addApp(AppDetail app){contextApps.add(app);}
```
Comment (hgg) [inline app/src/main/java/corp/kairos/adamastor/UserContext.java]: Shouldn't these methods be public so it's possible to add/remove/check apps from outside the class?

File: app/src/main/java/corp/kairos/adamastor/UserContext.java [hunk 3]
```diff
@@ -15,17 +21,49 @@
 public class UserContext {
     private String contextName; //The name of the context
     private List<AppDetail> contextApps; //The list of apps associated with this context
+    private Location location;
+    private GregorianCalendar init;
```
Comment (hgg) [inline app/src/main/java/corp/kairos/adamastor/UserContext.java:25]: We're using the GregorianCalendar for begin and end hour, is that right?

**Review by hgg — CHANGES_REQUESTED** [review]
Just left you a question regarding some of the class' variables and the access modifiers on 3 methods.

File: app/src/main/java/corp/kairos/adamastor/UserContext.java (hunk 3 above)
Comment (dfvt) [reply app/src/main/java/corp/kairos/adamastor/UserContext.java:25]: Yes

**Review by hgg — APPROVED** [review]

**Pull request merged**